standalone_app/src/sqlite3.ts

```diff
@@ -389,11 +389,11 @@ const getTrialIntermediateValues = (
           step: vals[0],
           value:
             vals[2] === "INF_NEG"
-              ? "-inf"
```

Comment:
Do we also need to change Line 221--Lline223, which is for multi-objective values?